rllib/common.py

```diff
@@ -216,7 +218,7 @@ class CLIArguments:
 
     # Train arguments
     # __cli_train_start__
-    Stop = typer.Option(None, "--stop", "-s", help=get_help("stop"))
+    Stop = typer.Option(None, *STOP_ARG_NAMES, help=get_help("stop"))
```

Comment:
@ArturNiederfahrenhorst hmm, I think you can fix all of this by making the default arg `"{}"` instead of `None`